src/rantanplan/structures.py

```diff
@@ -11,224 +10,145 @@
 # )
 # Structures will be checked in order of definition, the first one to match
 # will be chosen.
+ASSONANT_RHYME = "assonant"
+CONSONANT_RHYME = "consonant"
+ARTE_MAYOR_MIN_LENGTH = 9
+ARTE_MENOR_MAX_LENGTH = 8
+COPLA_ARTE_MENOR_MIN_LENGTH = 4
+LIRA_LONG_LINE = 11
+LIRA_SHORT_LINE = 7
+SILVA_LONG_LINE = LIRA_LONG_LINE
+SILVA_SHORT_LINE = LIRA_SHORT_LINE
+HENDECASYLLABLE = 11
+ALEXANDRINE = 14
+OCTOSYLLABLE = ARTE_MENOR_MAX_LENGTH
+TETRASYLLABLE = COPLA_ARTE_MENOR_MIN_LENGTH
+MINIMUM_SAFE_LENGTH = 3
+MAXIMUM_SAFE_LENGTH = 20
+STRUCTURES_LENGTH = {
+    "seguidilla_compuesta": [7, 5, 7, 5, 5, 7, 5],
+    "seguidilla": [7, 5, 7, 5],
+    "chamberga": [7, 5, 7, 5, 3, 7, 3, 7, 3, 7],
+    "seguidilla_gitana": [6, 6, 11, 6],
+    "estrofa_safica": [11, 11, 11, 5],
+    "estrofa_unamuno": [11, 11, 7, 5],
+    "estrofa_francisco_de_la_torre": [11, 11, 11, 7],
+    "endecha_real": [7, 7, 7, 11],  # puede repetirse N veces, no sirve así
```

Comment:
We could add some extra info to these definitions to indicate the repetition feature,
```suggestion
    "endecha_real": ([7, 7, 7, 11], True)  # here True could indicate that it could be repeated n times
```

Reply:
Yes I will work on that on the next PR

Reply:
Please, open an issue so we don't forget.